bigtable/unit_tests/test_retry.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+import subprocess, os, sys
+
+from google.cloud.bigtable.client import Client
+from google.cloud.bigtable.instance import Instance
+from google.cloud.bigtable.table import Table
+
+class TestRetry(unittest.TestCase):
+
+    TEST_SCRIPT = "unit_tests/retry_test.txt"
+
+    def test_retry(self):
+    	table, server = self.connect_to_server()
+        f = open(self.TEST_SCRIPT, 'r')
+        for line in f.readlines():
+            if line.startswith("CLIENT:"):
+                self.process_line(table, line)
+    	server.kill()
+
+    def process_line(self, table, line):
+        chunks = line.split(" ")
+        op = chunks[1]
+        if (op == "READ"):
+            self.process_read(table, chunks[2])
+        elif (op == "WRITE"):
+            self.process_write(table, chunks[2])
+        elif (op == "SCAN"):
+            self.process_scan(table, chunks[2], chunks[3])
+
+    def process_read(self, table, payload):
+        pass
+
+    def process_write(self, table, payload):
+        pass
+
+    def process_scan(self, table, range, ids):
```

Comment:
Looks like this test is a work in progress, but 2 things that jumped out as TODOs are to  process the stdout of the server (looking for PASS or FAIL) and to validate that the rows returned are correct according to the test script.